src/phases/move-phase.ts

```diff
@@ -299,6 +308,35 @@ export class MovePhase extends BattlePhase {
     }
   }
 
+  /** Queues a {@linkcode MoveChargePhase} for this phase's invoked move. */
+  protected chargeMove() {
+    const move = this.move.getMove();
+    const targets = this.getActiveTargetPokemon();
+
+    if (move.applyConditions(this.pokemon, targets[0], move)) {
+      // Protean and Libero apply on the charging turn of charge moves
+      applyPreAttackAbAttrs(PokemonTypeChangeAbAttr, this.pokemon, null, this.move.getMove());
+
+      this.showMoveText();
```

Comment:
```suggestion

    this.showMoveText();
    if (move.applyConditions(this.pokemon, targets[0], move)) {
      // Protean and Libero apply on the charging turn of charge moves
      applyPreAttackAbAttrs(PokemonTypeChangeAbAttr, this.pokemon, null, this.move.getMove());

```
So that if the move fails, it shows "Pokemon used Move!" before "But it failed!"